langkit/support/langkit_support-token_data_handlers.ads

```diff
@@ -285,4 +294,31 @@ package Langkit_Support.Token_Data_Handlers is
      (TDH : Token_Data_Handler; Line_Number : Positive) return Text_Type;
    --  Get the source text of line at index ``Line_Number``
 
+   function Get_Sloc
+     (TDH : Token_Data_Handler; Offset : Natural) return Source_Location;
+   --  Return the sloc for given ``Offset`` in ``TDH``. If:
+   --
+   --  - ``Offset`` is ``0``, return ``No_Source_Location``.
+   --
+   --  - ``Offset`` is in range ``1 .. Buffer'Last + 1``, return a
```

Comment:
s/Buffer/TDH.Source_Buffer/ (here and below), or say above that Buffer is TDH.Source_Buffer